pages/races/index.vue

```diff
@@ -1,25 +1,16 @@
 <template>
   <section class="docs-container container">
-    <h1>Races</h1>
-    <div
-      v-if="races && races.length == 0"
-      class="flex w-full flex-wrap pt-2 text-lg"
-    >
-      <div class="flex w-full">
-        There are no items for this category that align with the corresponding
-        sources you selected.
-      </div>
-      <div class="flex w-full pt-2">
-        Please edit your selected sources for more results.
-      </div>
+    <div class="filter-header-wrapper">
+      <h1 class="filter-header">Races</h1>
     </div>
-    <api-results-table :data="races" endpoint="races" />
+    <api-results-table
+      endpoint="races"
+      :api-endpoint="API_ENDPOINTS.sections"
+      :cols="['document__title', 'document__slug']"
+    />
```

Comment:
Currently this is returning a list of sections instead of a list of races. Does it need a filter like the other `sections` related items have?